pkpdapp/pkpdapp/urls.py

```diff
@@ -23,15 +22,78 @@
     path('admin/doc/', include('django.contrib.admindocs.urls')),
     path('admin/', admin.site.urls),
     path('accounts/', include('django.contrib.auth.urls')),
+
     path('dataset/<int:pk>/',
          views.DatasetDetailView.as_view(), name='dataset-detail'),
+
+    path(
+        'dataset/list/',
+        views.DatasetListView.as_view(),
+        name='dataset-list'
+    ),
+    path('dataset/add/', views.DatasetCreate.as_view(), name='dataset-add'),
+    path('dataset/create/', views.dataset.create, name='dataset-create'),
```

Comment:
I've been using "add" rather than "create", I don't mind what we settle on, but should be consistent